diff.py

```diff
@@ -865,9 +935,7 @@ def do_diff(basedump: str, mydump: str) -> List[OutputLine]:
             line_color1 = line_color2 = sym_color = Fore.RESET
             line_prefix = " "
             if line1 and line2 and line1.diff_row == line2.diff_row:
-                if maybe_normalize_large_imms(
-                    line1.original
-                ) == maybe_normalize_large_imms(line2.original):
+                if normalizer1.normalize(line1) == normalizer2.normalize(line2):
```

Comment:
hm, another thing. The normalizer is stateful, but that's not at all clear from this caller. I could see accidentally adding more calls to normalize(), and even now normalize() is only sometimes called for each line, while it probably should always be. Should this move into run_objdump maybe, adding another member to Line?

Reply:
Right, that would be less error prone. I've added a "normalized_original" attribute to Line and moved the normalizer to `process`